pkg/kn/commands/configuration_edit_flags.go

```diff
@@ -49,6 +50,7 @@ func (p *ConfigurationEditFlags) AddUpdateFlags(command *cobra.Command) {
 
 func (p *ConfigurationEditFlags) AddCreateFlags(command *cobra.Command) {
 	p.AddUpdateFlags(command)
+	command.Flags().BoolVar(&p.ForceCreate, "force", false, "Create service forcefully, replaces existing service if any.")
```

Comment:
"Overwrite an existing service with the same name instead of throwing an error"

Something liket that, because I think it's important to mention, what a "existing service" means (i.e. an existing service with the _same_ name)